### PR TITLE
LPS-147807 [CP 2.0] Improve how roles are disabled when user reaches role limit - remove Team Member button logic

### PR DESCRIPTION
**Pull request by claricebrasil**
### How it was developed?
Was created a button to the user be able to remove a line on Teams Member List. We used a property of Formik with FieldArray component to remove the last item of the list. Also we have to create a logic to update the numbers of Admin roles when the user remove a member team that has Administrator or Requestor role.

### What has been affected by the development?
We have to change the logic to add a new line on Teams Member to use the same property of Formik that we used to remove, but instead of use pop() to remove, we use push() (we were using the setState in the onClick.

### CODE REVIEW

**Review by Lucas2691 — APPROVED** [review]
Nice

**Review by joanagrego — APPROVED** [review]
Looks good to me!

**Review by thaynam — APPROVED** [review]

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js [hunk 1]
```diff
@@ -181,8 +181,9 @@ const InviteTeamMembersPage = ({
 					}))
 				);
 			}
-
 			setAvailableAdminsRoles(remainingAdmins);
+
+			return;
```
[the remark this answers is not in the document]
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js:186]: Ok, done!

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**